packages/ag-charts-community/src/chart/mapping/defaults.ts

```diff
@@ -32,26 +29,3 @@ export function swapAxes<T extends AgChartOptions>(opts: T): T {
         ],
     };
 }
-
-type PossibleObject = { enabled?: boolean } | undefined;
-
-export function resolveModuleConflicts<T extends AgChartOptions>(opts: T) {
```

Comment:
Where has this check moved to?

Reply:
https://github.com/ag-grid/ag-charts/pull/842/files/c9663906e8344ec265cc2b1ec00cedf3b2998dba#diff-08278e1c316b3be34805c99a6ac7216e4dc5a2d551ec152a508da7a58346e2c7R494